src/devices_models/devices/electric_loads.jl

```diff
@@ -4,44 +4,31 @@ struct StaticPowerLoad <: AbstractLoadFormulation end
 struct InterruptiblePowerLoad <: AbstractControllablePowerLoadFormulation end
 struct DispatchablePowerLoad <: AbstractControllablePowerLoadFormulation end
 
-########################### dispatchable load variables ####################################
-function AddVariableSpec(
-    ::Type{T},
-    ::Type{U},
-    ::PSIContainer,
-) where {T <: ActivePowerVariable, U <: PSY.ElectricLoad}
-    return AddVariableSpec(;
-        variable_name = make_variable_name(T, U),
-        binary = false,
-        expression_name = :nodal_balance_active,
-        sign = -1.0,
-        lb_value_func = x -> 0.0,
-        ub_value_func = x -> PSY.get_max_active_power(x),
-    )
-end
+########################### ElectricLoad ####################################
 
-function AddVariableSpec(
-    ::Type{T},
-    ::Type{U},
-    ::PSIContainer,
-) where {T <: ReactivePowerVariable, U <: PSY.ElectricLoad}
-    return AddVariableSpec(;
-        variable_name = make_variable_name(T, U),
-        binary = false,
-        expression_name = :nodal_balance_reactive,
-        sign = -1.0,
-        lb_value_func = x -> 0.0,
-        ub_value_func = x -> PSY.get_max_reactive_power(x),
-    )
-end
+get_variable_sign(_, ::Type{<:PSY.ElectricLoad}) = -1.0
```

Comment:
we have been using empty spaces for this. If we are moving into this style is important to document it and make it consistent.

Reply:
Can you clarify what you've been using empty spaces for?

Reply:
when the argument isn't used in the specific function for instance 

```julia
function foo(val::Int, other_val::Int)
return val + other_val
end

function foo(val::Int, ::Any)
return val
end
```

Reply:
I believe we discussed that using `_` is less noisy that using `::Any`. Can someone refresh my memory?

Reply:
I'm not sure that we reached consensus. I have no idea what Julia people would say. Here are my fairly weak opinions. I would go along with anything.
- In general passing arguments to functions that don't get used can be indicative of a sub-optimal design. In the cases here I think it makes sense. Hopefully, we don't have this pattern in many places.
- If it is a one-line function then `_` is less noisy.
- If it is a multi-line function then including the name of the variable at least informs the reader of what is being passed (to match the interface).

```
foo(val::Int, _) = val

function foo(val::Int, other_val)
    return val
end
```

Reply:
Just make sure that all the code base is consistent. I prefer not to use _ and pass the expected type.

Reply:
> If it is a one-line function then _ is less noisy.

I agree with this. Let me made this change for just one line functions.

Reply:
Okay, it is only implemented for one line functions.

Reply:
> In general passing arguments to functions that don't get used can be indicative of a sub-optimal design.

These methods are specialized. Here's two examples:

Call site:

```julia
lb = get_variable_upper_bound(variable_type, d, psi_container.settings)
...
init = get_variable_initial_value(variable_type, d, psi_container.settings)
```

For `get_variable_initial_value` the function is defined like this:

```julia
get_variable_initial_value(pv::ActivePowerVariable, d::PSY.ThermalMultiStart, settings) =
    get_variable_initial_value(pv, d, get_warm_start(settings) ? WarmStartVariable() : ColdStartVariable())
get_variable_initial_value(::ActivePowerVariable, d::PSY.ThermalMultiStart, ::WarmStartVariable) = PSY.get_active_power(d)
get_variable_initial_value(::ActivePowerVariable, d::PSY.ThermalMultiStart, ::ColdStartVariable) = nothing
```

For `get_variable_upper_bound` the function is defined like this:

```julia
get_variable_upper_bound(::ActivePowerVariable, d::PSY.ThermalMultiStart, _) = PSY.get_active_power_limits(d).max
```

My thinking here is that:

1) `settings` is untyped in `get_variable_initial_value` so it should be untyped in `get_variable_upper_bound`. 
2) `::WarmStartVariable` / `::ColdStartVariable` is used for dispatch in `get_variable_initial_value`. We leave out the variable name here because it doesn't matter, and I'm doing the same for `get_variable_upper_bound`.

Reply:
I have one last point that I have to make on this. I think it is not clear what the name of this variable should be. At the call site, we might be passing in a formulation, an instance of a formulation, the settings object etc. That is another reason I think using an underscore makes the most sense.

Reply:
I think we can form a consensus on the use of `_` in this fashion in a future PR. I'll go ahead and merge this for now.